File: swagger/models/get_appointment_options_response.py
from .base_model import BaseModel

from .appointment_option import AppointmentOption

class GetAppointmentOptionsResponse(BaseModel):
	"""
	no description available

	Attributes:

		options
		Contains information about the appointment options.
	"""

	input_map = {
		'Options': ['options', AppointmentOption],
		}

	output_map = {
		'options': 'Options',
		}

	options = None

File: swagger/Models/GetAppointmentOptionsResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetAppointmentOptionsResponse extends BaseModel
{
	/**
	 * Contains information about the appointment options.
	 * 
	 * @var AppointmentOption[] 
	 */
	public $options;

	protected function getInputMap()
	{
		return [
			'Options' => ['options', 'array', AppointmentOption::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'options' => ['Options', 'array', AppointmentOption::class],
			];
	}
}
